packages/models-library/src/models_library/api_schemas_webserver/product.py

```diff
@@ -16,12 +16,25 @@ class GetCreditPrice(OutputSchema):
         description="Price of a credit in USD. "
         "If None, then this product's price is UNDEFINED",
     )
+    min_payment_amount_usd: NonNegativeInt | None = Field(
```

Comment:
I would have computed the minimal amount in the most valuable currency on our planet: Osparc credits. That way if we one day want to switch to using, say, bitcoins instead of usd there is only 1 place where we need to change the conversion.